common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -229,7 +233,7 @@ def test_cant_submit_blank_answer(self):
             lambda: self.problem_page.problem_name == self.problem_name,
             "Make sure the correct problem is on the page"
         )
-        self.assertIn('is-disabled', self.problem_page.q(css='div.problem button.check').attrs('class')[0])
+        self.assertIn('is-disabled', self.problem_page.q(css='.problem .submit').attrs('class')[0])
```

Comment:
Same comment here-- use the helper method that will verify that both is-disabled is not present, and the disabled attribute is not present.